assets/js/copy-code-blocks.js

```diff
@@ -0,0 +1,40 @@
+// copy-code-blocks.js
+// Copyright: 2023  Scientific Python Developers
+// Author: Adam Porter <adam@alphapapa.net>
+// License: BSD 3-Clause License
+//   see <https://raw.githubusercontent.com/scientific-python/scientific-python-hugo-theme/main/LICENSE>
+
+// This file implements a button attached to code blocks which copies
+// the code to the clipboard.
+
+// This selector should match the PRE elements, each of which may
+// itself contain the text to be copied, or may contain a CODE element
+// which contains the text to be copied.
+const pre_selector = "div.highlight pre";
```

Comment:
Double check that this also catches fenced code blocks that don't specify a language.

Reply:
Yes, testing on all the code blocks on the `shortcodes/` page, ones both with and without a language are copied correctly.  Thanks.